docs/usage/key-concepts/scheduling.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Renovate scheduling
+description: Learn how to schedule when Renovate bot runs
+---
+
+This document describes Renovate bot's scheduling.
```

Comment:
Are we consistent about using "Renovate's" vs "Renovate bot's" in our docs? I thought we tended to use Renovate's

Reply:
As a "gut feeling" I'd say we use `Renovate's` more often than `Renovate bot's`, but I'll go check!

Reply:
I have checked and we use `Renovate's` everywhere, except in this PR. 😄 
I'll make a commit to fixup this sentence.